src/payment/strategies/googlepay/googlepay-checkoutcom-initializer.ts

```diff
@@ -0,0 +1,97 @@
+import { round } from 'lodash';
+
+import { Checkout } from '../../../checkout';
+import { InvalidArgumentError } from '../../../common/error/errors';
+import PaymentMethod from '../../payment-method';
+
+import { GooglePaymentData,
+    GooglePayInitializer,
+    GooglePayPaymentDataRequestV2,
+    TokenizePayload } from './googlepay';
+
+export default class GooglePayCheckoutcomInitializer implements GooglePayInitializer {
+
+    initialize(
+        checkout: Checkout,
+        paymentMethod: PaymentMethod,
+        hasShippingAddress: boolean
+    ): Promise<GooglePayPaymentDataRequestV2> {
+        return Promise.resolve(this._mapGooglePayCheckoutcomDataRequestToGooglePayDataRequestV2(
```

Comment:
If you mark `initialize` as `async`, you can just do a simple `return` and it'd be wrapped inside a `Promise`.

Reply:
changed as suggested